fdbserver/ptxn/TLogServer.actor.cpp

```diff
@@ -1516,4 +1659,28 @@ TEST_CASE("/fdbserver/ptxn/test/run_tlog_server") {
 	return Void();
 }
 
+TEST_CASE("/fdbserver/ptxn/test/peek_tlog_server") {
+	state test::TestDriverOptions options(params);
+	state std::vector<Future<Void>> actors;
+	state std::shared_ptr<test::TestDriverContext> context = test::initTestDriverContext(options);
+
+	for (const auto& group : context->tLogGroups) {
+		std::cout << "TLog Group " << group.logGroupId;
+		for (const auto& [storageTeamId, tags] : group.storageTeams) {
+			std::cout << ", SS team " << storageTeamId;
+		}
+		std::cout << "\n";
+	}
+
+	state std::string folder = "simdb/unittests";
```

Comment:
we need to create a random named folder so that each run is independent to each other. See "run_tlog_server" test case for example.

Reply:
Good idea! Done.